gemfiles/rails3.2.gemfile

```diff
@@ -3,3 +3,4 @@ source "https://rubygems.org"
 gemspec :path=>"../"
 
 gem "actionmailer", "~> 3.2.18"
+gem "test-unit"
```

Comment:
That the one that gets prompted now when trying to run rails 3.2 and ruby 2.2...